integration/tests/main_test.go

```diff
@@ -0,0 +1,41 @@
+// +build integration
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	openapi "github.com/asecurityteam/asset-inventory-api/client"
+)
+
+var assetInventoryAPI *openapi.APIClient //this contains pre-configured API client used by all fixtures
+
+func TestMain(m *testing.M) {
```

Comment:
nit: rename to TestSchemaVersion or something better?

Reply:
https://golang.org/pkg/testing/#hdr-Main . This is golang entrypoint for test run. Think main(). Magic name, so can not rename :(